src/accordion/Accordion.js

```diff
@@ -0,0 +1,78 @@
+import React, { cloneElement, useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const Accordion = ({ children, className, closedClassName, defaultActiveItems, onItemToggling, multiple, ...rest }) => {
+    const [activeItems, setActiveItems] = useState(defaultActiveItems);
+
+    const toggleItemActive = useCallback((index) => {
+        const isActive = activeItems.includes(index);
```

Comment:
Nitpick: I'd prefer `currentlyActive` as the variable name, as it something that will be changed.

Reply:
I will keep it just to be consistent with the `isActive` prop in the `AccordionItem`.